EnglishFilter/sections/general_extensions.txt

```diff
@@ -370,6 +370,8 @@ rosefile.net#%#//scriptlet("prevent-window-open")
 streamtape.com,strcloud.link#%#AG_onLoad(function() {setTimeout(function(){ var e = document.querySelectorAll(".plyr-overlay")[0]; if(e){e.click(); e.click();} }, 500) });
 ! https://github.com/AdguardTeam/AdguardFilters/issues/88646
 owllink.net#%#//scriptlet("prevent-window-open")
+! https://github.com/AdguardTeam/AdguardFilters/issues/95244
+owllink.net#%#//scriptlet('prevent-eval-if', 'adblockers')
```

Comment:
@marysaotomeprog 
This is a very popular anti-adblock "blockadblock" and we have a  pattern how to get rid of it. Just add the domain to:
```
#%#//scriptlet("prevent-bab")
$$script[wildcard="*eval*babasbmsgx*"][min-length="5000"][max-length="20000"]
```
and 
`@@||owllink.net^$generichide` f.e. under this rule `@@||voyeur-house.fun^$generichide` in content_blocker.txt
for platforms which don't support scriptlets and HTML-rules

Reply:
https://github.com/AdguardTeam/AdguardFilters/commit/dba51ea8c55e5a500ec1b00615baaedf097e00ae

Reply:
`slavaleleka` alsor recommend start from `abort-current-inline-scritpt` later `eval-if`